api/src/pcapi/notifications/push/backends/batch.py

```diff
@@ -87,11 +89,14 @@ def make_post_request(api: BatchAPI) -> None:
         make_post_request(BatchAPI.ANDROID)
         make_post_request(BatchAPI.IOS)
 
-    def send_transactional_notification(self, notification_data: TransactionalNotificationData) -> None:
+    def send_transactional_notification(
+        self, notification_data: TransactionalNotificationData, has_asynchronous_retry: bool = False
+    ) -> None:
         def make_post_request(api: BatchAPI) -> None:
             try:
                 response = requests.post(
                     f"{settings.BATCH_API_URL}/1.1/{api.value}/transactional/send",
+                    disable_synchronous_retry=has_asynchronous_retry,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Ça ne devrait pas être ` disable_synchronous_retry=not has_asynchronous_retry,` ?

Sinon j'ai compris l'inverse :thinking:

Reply:
Non : si tu as du asynchronous_retry (i.e tu es dans une CloudTask), alors tu ne veux pas avoir du retry synchrone fait dans `requests.py` (donc tu disable), car c'est la cloud task qui va retry

Reply:
Ok très clair !